### ISSUE TITLE
Event path & steps to dispatch an event needs to be clearly defined

### ISSUE THREAD

**Issue by rniwa** [a11y-api/a11y-api#103]
https://wicg.github.io/aom/spec/phase2.html says:
> Accessibility input events go through capture and bubble phases, just like DOM events. The only difference is that the capture and bubble phases happen entirely in the accessibility tree. If the event is not canceled (by calling preventDefault()) after completely going through the capture and bubble phases in the accessibility tree, the event executes its fallback behavior, as defined in the accessibility attribute table.
>
> When an event's fallback behavior involves firing a DOM event, the event's target will be the associated DOM node of the target AccessibleNode, or the associated DOM node of the nearest ancestor of the target AccessibleNode that has one. Fallback DOM events will go through capture and bubble phases just like any other DOM events.

This is not precise enough. The details of the event dispatching must be specified in as much detail as the one depicted in https://dom.spec.whatwg.org/#dispatching-events.

**Comment by rniwa** [comment]
@domenic @annevk

**Comment by annevk** [comment]
Yeah, you basically want to define "get the parent" for `AccessibleNode` objects I suppose so that when you dispatch an event on them they can go through the `AccessibleNode` tree.

**Comment by videetssinghai** [comment]
Sir, Can anyone label 2-3 issues as easy or good-first-issue.....I want to contribute to this project.

Thank you
